packages/v4/package.json

```diff
@@ -18,8 +18,8 @@
   },
   "dependencies": {
     "@patternfly/react-docs": "5.63.17",
-    "react": "^16.8.0",
```

Comment:
Let's update these to match what's in `@patternfly-react/react-core` so we're not removing support for react & react-dom 16.8+
- https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/package.json#L60-L61